cli/src/commands/init/index.ts

```diff
@@ -75,6 +75,7 @@ export default class Init extends BaseCommand {
       message: 'Do you want to install the TypeScript/JavaScript SDK?',
       initial: true
     });
+    if (confirm === undefined) return this.exit(1);
```

Comment:
We should add a linter for this

Reply:
Yeah, or a method in the base class I guess.